src/main/scala/nl/knaw/dans/easy/fedoratobag/EasyFedoraToBagApp.scala

```diff
@@ -222,6 +222,7 @@ class EasyFedoraToBagApp(configuration: Configuration) extends DebugEnhancedLogg
       _ = trace("created DDM from EMD")
       maybeFilterViolations <- options.datasetFilter.violations(emd, ddm, amd, fedoraIDs, allFileInfos)
       _ = if (options.strict) maybeFilterViolations.foreach(msg => throw InvalidTransformationException(msg))
+      _ = if ((ddm \\ "implemented").filter(_.prefix == "not").nonEmpty) throw InvalidTransformationException("not:implemented in the DDM")
```

Comment:
The `<not:imlpemented>` tags usually have some content. Please show that content in the error message to simplify trouble shooting.